docs/cli.md

```diff
@@ -0,0 +1,375 @@
+# Command Line Interface
```

Comment:
(my review is skipping the manual.md split-out-to-files and mkdocs-materialify because I think I'd risk -creep to enhance the content and this PR isn't for that)